odoo/addons/base/models/ir_cron.py

```diff
@@ -64,11 +67,13 @@ class ir_cron(models.Model):
                                       ('days', 'Days'),
                                       ('weeks', 'Weeks'),
                                       ('months', 'Months')], string='Interval Unit', default='months')
-    numbercall = fields.Integer(string='Number of Calls', default=1, help='How many times the method is called,\na negative number indicates no limit.')
-    doall = fields.Boolean(string='Repeat Missed', help="Specify if missed occurrences should be executed when the server restarts.")
+    numbercall = fields.Integer(string='NUMBER CALL DEPRECATED (KEPT FOR NO RUNBOT FAILED ATM)')
+    doall = fields.Boolean(string='DOALL DEPRECATED (KEPT FOR NO RUNBOT FAILED ATM)')
```

Comment:
As said, either remove the fields, either deprecate them (with deprecation warnings inside `_process_jobs?` but please don't change those strings. They are visible on the form view, if we were to merge the PR as-is, the customers would be very confused x)

Reply:
The fields are gonna be removed, but as I'll go through all cron to add the new features I will remove them every where in those PR and do the upgrade script after that

Reply:
It allows to keep this PR with a decent-sized changes for the review